apps/src/code-studio/pd/application/principalApproval1920/PrincipalApproval1920Component.jsx

```diff
@@ -208,38 +211,38 @@ export default class PrincipalApproval1920Component extends LabeledFormComponent
   }
 
   renderImplementationSection() {
-    const question_label = (<span>To participate in Code.org’s {this.props.teacherApplication.course} Professional
+    const questionLabel = (<span>To participate in Code.org’s {this.props.teacherApplication.course} Professional
                   Learning Program, we require that this course be offered in one of the following
                   ways. Please select which option will be implemented at your school. Be sure
                   to <a href="https://docs.google.com/document/d/1nFp033SuO_BMR-Bkinrlp0Ti_s-XYQDsOc-UjqNdrGw/edit#heading=h.6s62vrpws18" target="_blank">
                   review the guidance on required number of hours here</a> prior to answering.</span>);
-    const other_label = "We will use a different implementation schedule. (Please Explain):";
+    const otherLabel = "We will use a different implementation schedule. (Please Explain):";
 
     if (this.props.teacherApplication.course === 'Computer Science Discoveries') {
       return this.radioButtonsWithAdditionalTextFieldsFor('csdImplementation', {
-          [other_label] : 'other'
+          [otherLabel] : 'other'
         },
-        {label: question_label}
+        {label: questionLabel}
       );
     } else if (this.props.teacherApplication.course === 'Computer Science Principles') {
       return this.radioButtonsWithAdditionalTextFieldsFor('cspImplementation', {
-          [other_label] : 'other'
+          [otherLabel] : 'other'
         },
-        {label: question_label}
+        {label: questionLabel}
       );
     }
   }
 
   render() {
+    const courseSuffix = this.props.teacherApplication.course === 'Computer Science Discoveries' ? 'csd' : 'csp';
     return (
       <FormGroup>
         <p>
-          Thank you for your support of computer science education! A teacher at your
-          school, {this.props.teacherApplication.name}, has applied to be a part of
-          Code.org’s Professional Learning Program to teach the {' '}
-          {this.props.teacherApplication.course} curriculum during the {YEAR} school
-          year. Your approval is required for the teacher’s application
-          to be considered.
+          A teacher at your school, {this.props.teacherApplication.name}, has applied to be a part
+          of <a href="https://code.org/educate/professional-learning-2019" target="_blank">Code.org’s Professional Learning Program</a>
+          {' '} in order to teach the <a href={`https://code.org/educate/${courseSuffix}`} target="_blank">{this.props.teacherApplication.course} curriculum</a>
+          {' '} during the {YEAR} school year. Your approval is required for the
```

Comment:
This seems odd. Do we need the extra space? The `{' '}` and the space after it? 

Also nit: move the `of` on line 242 to the previous line so the link stands on its own. Also please move the 2nd link on 243 to its own line. I find it more readable when the html tags are on their own line. It's easier to spot them.